### PR TITLE
feat: add request-offline-floating-license

### PR DESCRIPTION
**Pull request by muneebkq**
add request offline floating license

### CODE REVIEW

File: src/lexfloatclient.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ export class HostProductVersionFeatureFlag {
 export class LexFloatClient {
 	/**
 	 * Sets the product id of your application.
-	 *
+
```
Comment (ahmad-kemsan) [inline src/lexfloatclient.ts:61]: @muneebkq why this change?

File: src/lexfloatclient.ts [hunk 2]
```diff
@@ -304,6 +304,20 @@ export class LexFloatClient {
 		}
 	}
 
+	/**
+	 * Sends the request to lease the license from the LexFloatServer for offline usage.
+	 * The maximum value of lease duration is configured in the config.yml of LexFloatServer 
+	 *
+	 * @param {number} leaseDuration seconds to lease the license for
+	 * @throws {LexFloatClientException}
```
Comment (ahmad-kemsan) [inline src/lexfloatclient.ts:312]: @muneebkq Please refer to the header file docs for lease duration.

File: src/native/LexFloatClient.h [hunk 3]
```diff
@@ -232,16 +232,23 @@ LEXFLOATCLIENT_API int LF_CC GetFloatingClientMeterAttributeUses(CSTRTYPE name,
...
-    FUNCTION: RequestFloatingLicense()
+    FUNCTION: RequestOfflineFloatingLicense()
+
+    PURPOSE: Sends the request to lease the license from the LexFloatServer for offline usage.
 
-    PURPOSE: Sends the request to lease the license from the LexFloatServer.
+    The maximum value of lease duration is configured in the config.yml of LexFloatServer 
+
+    PARAMETERS:
+    * leaseDuration - value of the lease duration.
```
Comment (ahmad-kemsan) [inline src/native/LexFloatClient.h:244]: @muneebkq Why is the leaseDuration property added as param for RequestFloatingLicense(). There seems to be other docs mistakes as well. Please check the docs again.

File: src/native/LexFloatClient.h [hunk 4]
```diff
@@ -268,6 +275,18 @@ LEXFLOATCLIENT_API int LF_CC DropFloatingLicense();
 */
 LEXFLOATCLIENT_API int LF_CC HasFloatingLicense();
 
+/*
+    FUNCTION: RequestOfflineFloatingLicense()
+
+    PURPOSE: Sends the request to lease the license from the LexFloatServer for offline usage.
+
+    RETURN CODES: LF_OK, LF_FAIL, LF_E_PRODUCT_ID, LF_E_LICENSE_EXISTS, LF_E_HOST_URL,
```
Comment (ahmad-kemsan) [inline src/native/LexFloatClient.h:283]: @muneebkq Why is the parameter missing here?

**Review by ahmad-kemsan — CHANGES_REQUESTED** [review]
Please check the comments

**Comment by ahmad-kemsan** [comment]
@muneebkq There are some merge conflicts as well. Please address them

File: src/lexfloatclient.ts (hunk 2 above)
Comment (muneebkq) [reply src/lexfloatclient.ts:312]: value of the lease duration.

File: src/lexfloatclient.ts (hunk 1 above)
Comment (muneebkq) [reply src/lexfloatclient.ts:61]: removed

File: src/native/LexFloatClient.h (hunk 4 above)
Comment (muneebkq) [reply src/native/LexFloatClient.h:283]: jsdoc modified

File: src/native/LexFloatClient.h (hunk 3 above)
Comment (muneebkq) [reply src/native/LexFloatClient.h:244]: jsdoc modified
